README.md

```diff
@@ -157,6 +157,10 @@ class Page
 
   # default from a singleton value (boolean in this case)
   attribute :published, Boolean, :default => false
+  
+  # default from a singleton value (string in this case),
+  # that doesn't allow a nil value
+  attribute :author, String, :default => 'Mies', :allow_nil => false
```

Comment:
I don't think the example should use a `default`, because it's much less likely that you'd need to worry about `nil` if you've got a default.